projects/sw-offline-google-analytics/test/unit/replay-queued-requests.js

```diff
@@ -38,17 +39,24 @@ const testLogic = (initialUrls, expectedUrls, time, additionalParameters) => {
 
 describe('replay-queued-requests', () => {
   const urlPrefix = 'https://replay-queued-requests.com/';
+  const initialTimestamp = 1470405670000;
```

Comment:
Sure no worries :)

On Fri, Aug 5, 2016, 8:10 AM Jeffrey Posnick <notifications@github.com>
wrote:

> In
> projects/sw-offline-google-analytics/test/unit/replay-queued-requests.js
> <https://github.com/GoogleChrome/sw-helpers/pull/36#discussion_r73707974>:
>
> > @@ -38,17 +39,24 @@ const testLogic = (initialUrls, expectedUrls, time, additionalParameters) => {
> >
> >  describe('replay-queued-requests', () => {
> >    const urlPrefix = 'https://replay-queued-requests.com/';
> > +  const initialTimestamp = 1470405670000;
>
> It's an arbitrary 📅 from earlier this morning, but the exact value
> doesn't matter in the context of the test. I'll put in a comment to that
> effect.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/sw-helpers/pull/36/files/1bd9efa309ca46eb77475a1761b4f67456a9478e#r73707974>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOigNG9clr1KOPfRSlbCDpG6WnPMGHxks5qc1JrgaJpZM4JduQ4>
> .
>